src/lib/rest/httpRequestSend.cpp

```diff
@@ -517,8 +517,15 @@ int httpRequestSendWithCurl
   }
 
 
+  //
+  // FIXME: outgoingMsgSize is a signed int. Max size ~2GB. Might not be enough.
+  //        Might be a good idea to change it to 'unsigned long long'
+  //
+
+  //
   // Check if total outgoing message size is too big
-  if (outgoingMsgSize > MAX_DYN_MSG_SIZE)
```

Comment:
After this change, is MAX_DYN_MSG_SIZE still in use?

Reply:
Good point.
It is (in `src/lib/common/macroSubstitute.cpp`), but it should not be.
Will use the new variable there and remove the define of MAX_DYN_MSG_SIZE

Reply:
Fixed in 636d28252